p2p/src/test/java/bisq/network/p2p/storage/mocks/AppendOnlyDataStoreServiceFake.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.network.p2p.storage.mocks;
+
+import bisq.network.p2p.storage.P2PDataStorage;
+import bisq.network.p2p.storage.payload.PersistableNetworkPayload;
+import bisq.network.p2p.storage.persistence.AppendOnlyDataStoreService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AppendOnlyDataStoreServiceFake extends AppendOnlyDataStoreService {
```

Comment:
This just uses standard terms for test objects so developers can understand the behavior and contract.  I use Fake, Dummy, and Stub as defined here: https://martinfowler.com/articles/mocksArentStubs.html#TheDifferenceBetweenMocksAndStubs

Specifically, fakes have working implementations of the interface, but use optimizations to make them easy to use in tests. For these, an in-memory map is used instead of persistent storage.

Reply:
ah nice. some reference :)

@ripcurlx, @m52go we should compile a devs-guide with PR rules, formatting, naming conventions (julian's ref eg.), etc. sometimes...